### ISSUE TITLE
enforce at the API / DB level that a user can only have one membership in a group

### ISSUE THREAD

**Issue by joachim-n** [Gizra/og#326]
It appears to be possible to use ogMembershipManager->createMembership() to create more than 1 membership entity for the same user and group.

Is this something that should be blocked, either at the DB level or in the API?

**Comment by amitaibu** [comment]
Yes, it should be blocked. I guess both blocks would be nice - primary keys on the DB and a nice exception description in the API.
